packages/endomoat/src/index.js

```diff
@@ -3,19 +3,57 @@ import 'ses'
 export * from './policy-converter.js'
 export * from './constants.js'
 
+import { toEndoPolicy } from './policy-converter.js'
+
+import fs from 'fs'
+
+import { importLocation } from '@endo/compartment-mapper'
+
 // TODO: need function which accepts filepath and policy
 // use a hardcoded policy for now
+/**
+ *
+ * @type {import('@endo/compartment-mapper').ReadFn}
+ */
+const readPower = async (location) =>
+  fs.promises.readFile(new URL(location).pathname)
 
+/**
+ * @type {import('@endo/compartment-mapper').ExitModuleImportHook}
+ */
 export const importHook = async (specifier) => {
   const ns = await import(specifier)
-  return Object.freeze({
-    imports: [],
-    exports: Object.keys(ns),
-    execute: (moduleExports) => {
-      moduleExports.default = ns
-      Object.assign(moduleExports, ns)
+  return Object.freeze(
+    /** @type {import('ses').ThirdPartyStaticModuleInterface} */ ({
+      imports: [],
+      exports: Object.keys(ns),
+      execute: (moduleExports) => {
+        moduleExports.default = ns
+        Object.assign(moduleExports, ns)
+      },
+    })
+  )
+}
+
+/**
+ *
+ * @param {string} entrypointPath
+ * @param {import('lavamoat-core').LavaMoatPolicy} policy
+ */
+export const run = async (entrypointPath, policy) => {
+  const { namespace } = await importLocation(readPower, entrypointPath, {
+    policy: toEndoPolicy(policy),
+    globals: globalThis,
+    importHook,
+    modules: {
+      USE_ME_AS_DEFAULT_ATTENUATOR_NAME: {
+        attenuateGlobals: () => {
+          console.log('lol, this works')
+        },
+      },
```

Comment:
can we just zap this if we're not gonna use it, because having it there confuses my dumb monkey brain

Reply:
ok. 
I'm not sure if we're guaranteed not to use it, but I don't mind deleting. It's not like we'll forget it existed now :)